datarepo-clienttests/src/main/java/utils/KubernetesClientUtils.java

```diff
@@ -48,6 +48,10 @@
   private static CoreV1Api kubernetesClientCoreObject;
   private static AppsV1Api kubernetesClientAppsObject;
 
+  public enum PodPhase {
+    running
+  }
```

Comment:
I don't know if this is the same thing. But does [this](https://github.com/kubernetes-client/java/blob/295a00a909ef23989b6e58b075588099f556e6d5/kubernetes/src/main/java/io/kubernetes/client/openapi/models/V1ContainerState.java#L30-L31) match the container states/phases you see?

Reply:
I changed it to instead check that the correct number of pods are "ready," removing the need to check against a status. It also seems to be a little more accurate check.